controllers/territory_controller/tests.py

```diff
@@ -184,3 +187,85 @@ def test_BE_links(self) -> None:
             {StationCode.EY, StationCode.VB, StationCode.SZ, StationCode.PO},
             'Territory BE has incorrect territory links',
         )
+
+
+class TestTerritoryLockout(unittest.TestCase):
+    "Test that individual territories become 'locked' when claimed a set number of times"
+
+    _zone_0_territories = {StationCode.PN, StationCode.EY}
+    _zone_1_territories = {StationCode.YL, StationCode.PO}
+
+    def load_territory_owners(self, claim_log: ClaimLog) -> None:
+        # set territories owned by zone 0
+        for territory in self._zone_0_territories:
+            claim_log._station_statuses[territory] = Claimant.ZONE_0
+
+        # set territories owned by zone 1
+        for territory in self._zone_1_territories:
+            claim_log._station_statuses[territory] = Claimant.ZONE_1
+
+    @patch('controller.Supervisor.getFromDef')
+    @patch('territory_controller.get_robot_device')
+    def setUp(self, mock_get_robot: Mock, mock_getFromDef: Mock) -> None:
+        super().setUp()
+        claim_log = ClaimLog(record_arena_actions=False)
+        self.load_territory_owners(claim_log)
+        self.attached_territories = AttachedTerritories(claim_log)
+        with patch('territory_controller.TerritoryController.set_score_display'):
```

Comment:
Would be great to be consistent about how we're doing this patching -- either all as context managers or all as decorators.

Reply:
682cb6e0d4248ddaa454d0e4227e9ef15b3d8ed5